workers/generic-worker/taskcluster_proxy.go

```diff
@@ -57,12 +57,13 @@ func (l *TaskclusterProxyTask) RequiredScopes() scopes.Required {
 }
 
 func (l *TaskclusterProxyTask) Start() *CommandExecutionError {
-	if l.task.D2GInfo != nil {
-		out, err := exec.Command("docker", "network", "inspect", "bridge", "--format", "{{range .IPAM.Config}}{{if .Gateway}}{{.Gateway}} {{end}}{{end}}").Output()
+	switch l.task.Payload.TaskclusterProxyInterface {
+	case "docker-bridge":
+		out, err := host.CombinedOutput("docker", "network", "inspect", "bridge", "--format", "{{range .IPAM.Config}}{{if .Gateway}}{{.Gateway}} {{end}}{{end}}")
 		if err != nil {
 			return executionError(internalError, errored, fmt.Errorf("could not determine docker bridge IP address: %s", err))
 		}
-		gateways := strings.Split(strings.TrimSpace(string(out)), " ")
```

Comment:
I had initially suggested this in https://github.com/taskcluster/taskcluster/pull/7564#discussion_r1979583155 but @jcristau preferred the `exec.Command().Output()` method since he explicitly did not want stderr to be included here.

Reply:
Maybe we can make a new helper that you can get stdout and stderr separately assigned in one call? I think having docker's stderr in the `executionError` might be beneficial here.